pages/profile/[username].tsx

```diff
@@ -1,35 +1,37 @@
 import * as React from 'react'
+import { useQuery } from '@apollo/react-hooks'
 import _ from 'lodash'
 import Layout from '../../components/Layout'
+import { useRouter } from 'next/router'
 import { Lesson } from '../../@types/lesson'
-import { AppData } from '../../@types/app'
 import { UserSubmission, Challenge } from '../../@types/challenge'
-import GET_APP from '../../graphql/queries/getApp'
+import USER_INFO from '../../graphql/queries/userInfo'
```

Comment:
We should be importing from `graphql/index.tsx` only now, and using the generated types.

Reply:
Reference https://github.com/garageScript/c0d3-app/blob/4c3f4c0c6d89010fe708021d24f66b5c507f2df4/pages/index.tsx#L6